src/pages/payment-methods/_mobile-expanded-list.js

```diff
@@ -84,6 +86,9 @@ const StyledRefLink = styled(Flex)`
     @media ${device.mobileL} {
         justify-content: flex-start;
     }
+    @media ${device.tabletM} {
+        width: 65%;
+    }
```

Comment:
Can we can use Flex here and pass in most of these as props?

Reply:
@suthesan-deriv Hi , could you please elaborate more on this ? or any part of the code using it and I can refer to it? 


Also as another work around  I thought of removing the 50% width only In `StyledItemDiv` to let it be more responsive. like this 

<img width="461" alt="Screen Shot 2022-01-13 at 10 32 36 AM" src="https://user-images.githubusercontent.com/64970259/149281627-a34eba3a-5983-4986-a599-e7bf69f0ac80.png">

Please do let me know if there is a better way , thanks 🙏